apihub/subscription/models.py

```diff
@@ -15,19 +67,27 @@ class Subscription(Base):
     __tablename__ = "subscriptions"
 
     id = Column(Integer, primary_key=True, index=True)
-    application = Column(String, unique=True, index=True, nullable=False)
+    # duplicate tier to make it easier to query.
     tier = Column(Enum(SubscriptionTier), default=SubscriptionTier.TRIAL)
     active = Column(Boolean, default=True)
-    credit = Column(Integer, default=0)
+    # credit = Column(Integer, default=0)
```

Comment:
We can query balance/credit to get a percentage of spending, maybe keep it

Reply:
yes I undo it on other commit